rpm_docker.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sudo service docker start
+docker build -t centos:latest .
+docker cp `docker ps -l -q`:/root/rpmbuild/RPMS/x86_64/${PACKAGE}-${VERSION}-1.x86_64.rpm ./
```

Comment:
Could run `package_cloud push ~~`  in docker??
It is simple that rpm build and push by one liner `docker run` , I think.